package.json

```diff
@@ -10,9 +10,10 @@
 		"build-watch": "lerna run build-watch --parallel",
 		"clean": "lerna clean",
 		"publish": "lerna publish",
-		"test": "lerna run test --parallel",
+		"test": "lerna run test --parallel && npm run test-e2e",
 		"test-updated": "lerna run test --since --parallel",
 		"updated": "lerna updated",
+		"install-e2e": "cd e2e-tests && npm install",
```

Comment:
You should use learna `--scope` flog.

Reply:
I'm not sure I can use lerna here, I'm getting the following error:
```
No packages remain after filtering [ 'e2e-tests' ]
```

Reply:
You can :)

Reply:
Something not set up properly then.

Reply:
But why do you want a specific install for this one anyway? It should be included in the `bootstrap` script :)

Reply:
Same for the test?